dev/storage_base.h

```diff
@@ -47,19 +45,24 @@ namespace sqlite_orm {
             void drop_index(const std::string& indexName) {
                 std::stringstream ss;
                 ss << "DROP INDEX " << quote_identifier(indexName);
-                perform_void_exec(get_connection().get(), ss.str());
+                perform_void_exec(this->get_connection().get(), ss.str());
+            }
+
+            void drop_trigger(const std::string& triggerName) {
```

Comment:
@fnc12 This was a [templated method](https://github.com/fnc12/sqlite_orm/blob/451a81b774123d07f45413db4f4604aae0773fb1/dev/storage.h#L217) previously. Was there any reason for this?

Reply:
I am just wondering then why the other methods don't accept "any thing"?

Reply:
cause it is legacy.
yes `std::string_view` is a perfect solution but it is not available in C++14.

Reply:
I assume that `drop_trigger` came after `drop_index` and `drop_table`? If so, can we just adjust `drop_trigger` to the other methods, and possibly make a feature request?

Reply:
If somebody wants to pass a different thing than a std::string, that would be a feature request. Either we create that request ourselves or somebody in need.

Reply:
oh I see. Ok let's leave it like that. Probably we are discussing about something not used at all.